docs/reference/module-api/module-overview.md

```diff
@@ -98,6 +99,10 @@ An object used to add additional, often optional, settings to a module. There ar
 
 Set to `false` to prevent the module from being fully instantiated in the application. The primary purpose of this option is to create a base class module that others will [extend](#extend) but that will not be used directly.
 
+### `bundle`
+
+Used to add multiple modules from a single base module. Takes an object with two properties. The `directory` property takes a relative path to the directory of the modules to be loaded. The `modules` property takes an array of module names. Modules loaded in this way still need to be added to the `app.js` file, unless they use [`improve`](#improve). Any modules that to be used as a base class for other modules should be added to `app.js`, but have their [`instantiate`](#instantiate) option set to `false`.
```

Comment:
"from a single npm module." It must be an npm module and has no effect at all in a project level module, so let's make that clear from the get-go.

Reply:
"instantiate property", not "instantiate option" (it's not an option, it doesn't go under `options`).

Reply:
The original module must be activated in `app.js` as well if any of the other modules in the bundle are to be used. The original module can and usually does contain functionality of its own, in addition to delivering a bundle of additional modules.

Reply:
Altered test for all three comments.